src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -265,36 +264,20 @@ def get_context(group: Group) -> str:
     return context_text.rstrip()
 
 
-def get_option_groups(group: Group) -> Sequence[Mapping[str, Any]]:
-    all_members = group.project.get_members_as_rpc_users()
-    members = list({m.id: m for m in all_members}.values())
-    teams = group.project.teams.all()
-
-    option_groups = []
-    if teams:
-        option_groups.append({"text": "Teams", "options": format_actor_options(teams)})
-
-    if members:
-        option_groups.append({"text": "People", "options": format_actor_options(members)})
-
-    return option_groups
-
-
 def get_option_groups_block_kit(group: Group) -> Sequence[Mapping[str, Any]]:
     all_members = group.project.get_members_as_rpc_users()
     members = list({m.id: m for m in all_members}.values())
     teams = group.project.teams.all()
-    use_block_kit = features.has("organizations:slack-block-kit", group.project.organization)
 
     option_groups = []
     if teams:
-        team_options = format_actor_options(teams, use_block_kit)
+        team_options = format_actor_options(teams, True)
```

Comment:
next PR we can remove the second boolean parameter right?

Reply:
yes

Reply:
i'm trying to isolate the changes to this one file, but it ended up affecting 123098 tests